x-pack/plugins/index_lifecycle_management/public/application/services/find_errors.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { propertyof, ValidationErrors } from './policies/policy_validation';
+
+export const findFirstError = (errors?: ValidationErrors): string | undefined => {
```

Comment:
I would have put this handler inside the `policy_validation` file as it is related to policy. By putting it here and giving it a generic name "findFirstError" we might have issue later on if we have other validations.